content/language-agnostic/encyclopedia/what-is-a-terminal?.md

```diff
@@ -0,0 +1,22 @@
+<!--
+---
+Subjects:
+  - "What is a terminal?"
+Tags: 
+  - "language-agnostic"
+  - "bash/shell"
+  - "git"
+  - "command line"
+Catalog Content:  # If listing multiple URLs, please put the most relevant one first 
+  - "https://www.codecademy.com/learn/learn-the-command-line"
+---
+-->
+
+# What is a terminal?
+
+A terminal, also known as a command line, is a program which allows programmers to type commands directly to the internal CPU or another network device. It is a no-frills program where a user simply inputs text as opposed to a graphical user interface (GUI) where a user interacts with windows, buttons, and icons.
```

Comment:
I would take out ", also known as a command line, " just in case some hardcore programmers will roast us cus they are nitpicky about these terms. So:

A terminal is a program which allows...